config.go

```diff
@@ -4,6 +4,7 @@ import (
 	"net/http"
 
 	"github.com/gin-gonic/gin"
+	"strings"
```

Comment:
move to the top for the internal package.

Reply:
otherwise LGTM 👍

Reply:
done